src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/VisibilityAndInteraction/index.spec.tsx

```diff
@@ -139,6 +148,37 @@ describe('<VisibilityAndInteraction />', () => {
     await screen.findByText('Url copied in clipboard !');
   });
 
+  it('checks the lti link and copy the code in clipboard', async () => {
+    const mockedVideo = videoMockFactory({
+      is_public: true,
+    });
+
+    // simulate video update
+    render(
+      wrapInVideo(
+        <InfoWidgetModalProvider value={null}>
+          <VisibilityAndInteraction />
+        </InfoWidgetModalProvider>,
+        mockedVideo,
+      ),
+    );
+
+    const copyButtonReRendered = screen.getByRole('button', {
+      name: 'LTI link:',
+    });
+    expect(copyButtonReRendered).not.toBeDisabled();
+
+    screen.getByText('https://localhost/lti/videos/'.concat(mockedVideo.id));
```

Comment:
```suggestion
    expect(screen.getByText(`https://localhost/lti/videos/${mockedVideo.id}`).toBeInTheDocument();
```

Reply:
[fixup! ✨(lib-video) lti code to copy to clipboard](https://github.com/openfun/marsha/pull/2227/commits/c7fe7c29cfe901a9688f97fe304d50ecc1078044)